go.mod

```diff
@@ -1,6 +1,6 @@
 module sigs.k8s.io/kueue
 
-go 1.21
+go 1.21.6
```

Comment:
```suggestion
```

I'm not sure why we need this patch version.

Reply:
oh shoot, we shouldn't need it.

I can send a PR in a bit.

Reply:
One you submit PR, let me know.